config/management/operational/src/validator_set.rs

```diff
@@ -44,11 +45,24 @@ impl ValidatorSet {
                 Ok(config) => config,
                 Err(err) => {
                     println!(
-                        "Unable to decode account {}: {}",
+                        "Unable to decode account {}: {}. Using a dummy validator network address!",
                         info.account_address(),
                         err
                     );
-                    continue;
+
+                    // Return the partially decrypted validator config (using a dummy validator
+                    // network address).
+                    let encrypted_config = info.config();
+                    DecryptedValidatorConfig {
+                        name: "".to_string(),
+                        consensus_public_key: encrypted_config.consensus_public_key.clone(),
+                        fullnode_network_address: encrypted_config
+                            .fullnode_network_addresses()
+                            .map_err(|e| Error::NetworkAddressDecodeError(e.to_string()))?[0]
+                            .clone(),
+                        validator_network_address: NetworkAddress::from_str("/dns4/0000")
```

Comment:
@gregnazario, any idea for a better dummy validator network address? The DecryptedValidatorConfig object requires one, and I'd ideally like to avoid refactoring DecryptedValidatorConfig to also take in an encrypted adress in the case of a failure... (hence the dummy address here). Any better suggestions?

Reply:
`/dns4/could-not-decrypt` or, just make it an `Option<NetworkAddress>`

Reply:
Ooh, nice! Let's go with "could-not-decrypt" -- I'm in favour of displaying something in the output where the address should be so that the operator knows the decryption failed (e.g., if the validator set is long, they might miss the error lines at the top)